models/friend/index.js

```diff
@@ -199,11 +199,26 @@ async function getBlockedUsers({ id }) {
 async function blockUser(uId, notUId) {
   try {
     // needs to be changed to reflect situations where ids are swapped (see delete friend request query)
-    var queryString = 'UPDATE `request` SET `blocked`=1 '  +
-      'WHERE (`receiver_id`=? AND `sender_id`=?) ' +
-      'OR (`receiver_id`=? AND `sender_id`=?)';
-    const results = await DB.query(queryString,
-      [uId, notUId, notUId, uId ]);
+    var queryString = 'INSERT INTO `request` SET `receiver_id`=?, `sender_id`=?, `blocked`=1 ' +
+        'ON DUPLICATE KEY UPDATE `blocked`=1';
+    const results = await DB.query(queryString, [uId, notUId]);
+    const auxResults = await blockUserAux(uId, notUId);
+    return auxResults;
+  }  catch (error) {
+    Logger.error(error);
+    throw error;
+  }
+}
+
+async function blockUserAux(uId, notUId) {
+  try {
+    // needs to be changed to reflect situations where ids are swapped (see delete friend request query)
+    console.log(uId);
+    console.log(notUId);
+    var queryString = 'INSERT INTO `blocked` SET `blocker_id`=?, `blockee_id`=? ';
+    const results = await DB.query(queryString, [uId, notUId]);
+    Logger.error(uId);
```

Comment:
Should these be `error` or `debug`?

Reply:
From https://github.com/winstonjs/winston you should be able to do `Logger.info` or `Logger.debug` if needed

```
const levels = { 
  error: 0, 
  warn: 1, 
  info: 2, 
  verbose: 3, 
  debug: 4, 
  silly: 5 
}
```